### PR TITLE
Issue/49

### PR DESCRIPTION
**Pull request by titay2**

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -51,8 +53,18 @@ $goBackButton.on('click', function () {
...
+//
+// }
+})
 $updateAppForm.on('submit', function (event) {
   event.preventDefault()
+  //rendeupdateform()
+
   var changed = $('#rename-app').val()
   var app = {
     id: $('#name-app').data('id'),
```
Comment (gr2m) [inline scripts/main.js]: just as a reminder, the problem was with this line. `$('#name-app').data('id')` returned an empty string instead of the app id

**Comment by gr2m** [comment]
let me know if you want me to look into this PR or if you have any questions

**Comment by titay2** [comment]
@gr2m  I am still trying to see whats wrong but I cant figure this out it would be nice if you can help when ever you will have time :)

**Comment by gr2m** [comment]
@titay2 have you pushed the latest version of your code?

**Comment by titay2** [comment]
yes i did

**Comment by gr2m** [comment]
I’ve rebased the PR on `gh-pages` and `git push --force`d it, so before you do anything else on branch `issue/49`, make sure you pull the latest changes

**Comment by gr2m** [comment]
@titay2 I cleaned up the code a little bit and removed the styling of the modal. I had no problems with it, can you check it again? If it works for you, I’d say let’s merge it in?

One more thing I’d like to do with the code before moving over to riot.js is to add a route for editing an app. I’d suggest something like `/#abc4567/edit` for the app with id `abc4567`. In the `handleRoute` function where we check for the routes, we can then check for these routes with something like this

```
if (path.substr(-5) === '/edit') {
  var id = path.substr(0, path.length - 5)
  renderEditAppForm(id)
  return
}
```

And then implement `renderEditAppForm` similar to how `renderAppDetail` is implemented. 
Does that all make sense? But as I said, I’d suggest we merge this PR in, and then work on the route in a separate PR

**Comment by titay2** [comment]
@gr2m  thank you so much for the help :) but after i merged the two codes i can see the old code only on the gh-pages the changes from this branch has been omitted

**Pull request merged**